### DISCUSSION TITLE
Notify.run not available for days

### DISCUSSION THREAD

**Discussion by maxmuster001** [rizwansoaib/whatsapp-monitor#119] [General]
Notify.run not available for days

Error reaching API server.

No new channel is possible either

**Comment by rizwansoaib** [comment]
notify.run server is not created by me it's open source I just use it in
our application.



I will try to build web push notification server.


Thank You


On Tue, 19 Oct, 2021, 2:38 am maxmuster001, ***@***.***>
wrote:

> Notify.run not available for days
>
> Error reaching API server.
>
> No new channel is possible either
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/119>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZATHLJMFVGTWO4J3SLUHSEGHANCNFSM5GHSNJ2A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by maxmuster001** [reply]
Thank you

**Comment by rizwansoaib** [comment]
https://github.com/notify-run/notify.run/issues/34

**Comment by rizwansoaib** [comment]
notify.run is working now...

On Tue, 19 Oct, 2021, 8:58 pm maxmuster001, ***@***.***>
wrote:

> Thank you
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/119#discussioncomment-1503231>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZDPFIWXCJQM7OBC5GLUHWFCLANCNFSM5GHSNJ2A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.
